docs/specs/schema.yml

```diff
@@ -487,6 +487,150 @@ outputs:
+          }
+        },
+        "trophy": {
+          "type": "object",
+          "required": ["uid"],
+          "properties": {
+            "uid": {
+              "type": "string",
+              "contentType": "uid"
+            }
+          }
+        },
+        "achievement": {
+          "type": "string",
+          "contentType": "xref"
+        }
+      }
+    }
+outputs:
+  .errors.log: |
+    {"message_severity":"error","code":"path-uid-missing","message":"Missing attribute: UID. A valid UID is required for each learning path.","file":"patha.yml","line":2}
+    {"message_severity":"error","code":"path-modules-missing","message":"Missing attribute: modules. At least one valid module is required for each learning path.","file":"patha.yml","line":2}
+    {"message_severity":"error","code":"path-trophy-missing","message":"Missing attribute: trophy. A valid trophy is required for each learning path.","file":"patha.yml","line":2}
+    {"message_severity":"error","code":"path-child-not-found","message":"Child UID(s): 'module_not_found' can't be found.","file":"pathb.yml","line":3}
+---
+# error mapping for Learn Valdiation based on SDP, Module
+inputs:
+  docfx.yml: 
+  modulea.yml: |
+    #YamlMime:Module
+    a: a
+  moduleb.yml: |
+    #YamlMime:Module
+    uid: module_b
+    units: ['unit_not_found']
+    badge: {
+      uid: badge_b
+    }
+  _themes/ContentTemplate/schemas/Module.schema.json: |
+    {
+      "required": ["uid", "units"],
+      "either": [["badge", "achievement"]],
+      "properties": {
+        "uid": {
+          "type": "string",
+          "contentType": "uid"
+        },
+        "units": {
+          "type": "array",
+          "minItems": 1,
+          "items": {
+            "type": "string",
+            "contentType": "xref"
+          }
+        },
+        "badge": {
+          "type": "object",
+          "required": ["uid"],
+          "properties": {
+            "uid": {
+              "type": "string",
+              "contentType": "uid"
+            }
+          }
+        },
+        "achievement": {
+          "type": "string",
+          "contentType": "xref"
+        }
+      }
+    }
+outputs:
+  .errors.log: |
+    {"message_severity":"error","code":"module-uid-missing","message":"Missing attribute: UID. A valid UID is required for each module.","file":"modulea.yml","line":2}
+    {"message_severity":"error","code":"module-units-missing","message":"Missing attribute: units. At least one valid unit is required for each module.","file":"modulea.yml","line":2}
+    {"message_severity":"error","code":"module-badge-missing","message":"Missing attribute: badge. A valid badge is required for each module.","file":"modulea.yml","line":2}
+    {"message_severity":"error","code":"module-child-not-found","message":"Child UID(s): 'unit_not_found' can't be found.","file":"moduleb.yml","line":3}
+---
+# error mapping for Learn Valdiation based on SDP, ModuleUnit
+inputs:
```

Comment:
add a test case that property need to map in level 2

Reply:
OK.